src/libraries/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/TransformChain.cs

```diff
@@ -150,12 +150,12 @@ public int Count
             throw new CryptographicException(SR.Cryptography_Xml_TransformIncorrectInputType);
         }
 
-        internal Stream? TransformToOctetStream(Stream? input, XmlResolver resolver, string baseUri)
+        internal Stream? TransformToOctetStream(Stream? input, XmlResolver? resolver, string baseUri)
         {
             return TransformToOctetStream(input, typeof(Stream), resolver, baseUri);
         }
 
-        internal Stream? TransformToOctetStream(XmlDocument document, XmlResolver resolver, string baseUri)
+        internal Stream? TransformToOctetStream(XmlDocument document, XmlResolver? resolver, string baseUri)
         {
             return TransformToOctetStream(document, typeof(XmlDocument), resolver, baseUri);
         }
```

Comment:
`TransformToOctetStream(object,Type,XmlResolver,string)` already accepts the object as being null.

This method doesn't do anything special with document, just passes it into the null-accepting `inputObject` parameter.

So this method could accept document as being null, and get rid of some `!`s and `#pragma`s.

Reply:
Good spot! Addressed in [this commit](https://github.com/dotnet/runtime/pull/67198/commits/4b66823c3ec36765fbf57dc3853335d7f2ffc027).